### DISCUSSION TITLE
Building an Ethereum Gas Station in our services

### DISCUSSION THREAD

**Discussion by yisiliu** [DimensionDev/Maskbook#2376] [Ideas]
We can build a gas station to help ease the cost of transactions with Mask native smart contracts for our users. Is it a good timing to do so? My intuition is quite naive here but we can actually design a complicated (which I don't quite like) economic system upon it. What do you guys think?

**Comment by Sergey130** [comment]
Супер, что нужно для этого ?

**Reply by yisiliu** [reply]
since it'll help us ease the cost you need for interacting with our smart contracts

**Comment by dawafisher** [comment]
Good idea!
